tests/run.sh

```diff
@@ -6,8 +6,16 @@ k8s_version=$1
 istio_version=$2
 install_dir=$3
 
-source ./create_cluster.sh $k8s_version "virtualbox"
-./install_istio.sh $istio_version "osx"
+# Uncomment below to run integration test on local computer to spin up minikube
+#source ./create_cluster.sh $k8s_version "virtualbox"
```

Comment:
You can control this using a env variable like if `IS_LOCAL` == true then run this step. It would be cleaner in `Contributing.md` instructions